src/components/AgmExtension/AgmExtensionEvaluation.vue

```diff
@@ -51,7 +52,22 @@
           cols="12"
           sm="9"
         >
-          {{ data.extensionDuration || 'Unknown' }}
+          <template v-if="!data.isEligible">
+            <template v-if="data.alreadyExtended && !data.requestExpired">
+              The business has reached maximum possible extension for this AGM.
+            </template>
+            <template v-else-if="data.requestExpired && !data.alreadyExtended">
+              The period to request an extension has expired.
+            </template>
+            <template v-else-if="data.requestExpired && data.alreadyExtended">
+              The business has reached maximum possible extension for this AGM.
+              The period to request an extension has expired.
+            </template>
+          </template>
+          <template v-else>
+            {{ data.extensionDuration }} months
+          </template>
+        </v-col>
```

Comment:
I like it. My only request is, could you keep the variables in the same order in the conditionals?

Reply:
I caught that just now too 😄